sky/skylet/providers/lambda_cloud/node_provider.py

```diff
@@ -16,12 +16,14 @@
     NODE_KIND_WORKER,
     NODE_KIND_HEAD,
 )
-from ray.autoscaler._private.util import hash_launch_conf
 from sky.skylet.providers.lambda_cloud import lambda_utils
-from sky.utils import common_utils
+from sky import authentication as auth
+from sky.utils import command_runner
+from sky.utils import subprocess_utils
 
 TAG_PATH_PREFIX = '~/.sky/generated/lambda_cloud/metadata'
-REMOTE_RAY_YAML = '~/ray_bootstrap_config.yaml'
+REMOTE_RAY_SSH_KEY = '~/ray_bootstrap_key.pem'
+GET_INTERNAL_IP_CMD = 'ip -4 -br addr show | grep -Eo "10\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)"'
```

Comment:
Add prefix `_` to these variables as they are internal variables?

Reply:
Done.